src/dbnode/storage/repair.go

```diff
@@ -606,8 +606,9 @@ func (r *dbRepairer) Repair() error {
 
 			if err := r.repairNamespaceBlockstart(n, blockStart); err != nil {
```

Comment:
In the case where we hit an error, are we swallowing the error now? We should probably make sure that the error gets logged somewhere, even as we move on to the next block.

Reply:
Bleh nvm, bad reading on my part--error will still get propagated here.